README.md

```diff
@@ -1,2 +1,80 @@
 # paperTemplate
+
 Template repository for scientific journal articles and associated materials
+
+## Input data
+
+Use the `data-raw` folder to save code that extracts or downloads the input
+data used for your paper. This code can output data to the `data` directory.
+
+Use the `data` directory to store input data. In this template, `.rds` files
+in the `data` directory will not be tracked with git (see `.gitignore` file).
+
+## Analysis
+
+Use the `analysis` directory to save analysis code. Analysis should be split
+into separate scripts appropriately. Operations that are common across analysis
+scripts can be functionalized and saved to the `utils.R` file to minimize
+repeated code.
+
+Outputs of analysis can also be saved to the `data` directory as `.rds` files, 
+or a new `results` directory can be created, as needed.
+
+## Writing
+
+Rmarkdown files are useful for paper writing, because code can be embedded
+within, making number-plugging easier and more reproducible.
+
+This template includes three Rmarkdown files:
+1. manuscript
+2. tables_figures
+3. appendix
+
+It may be appropriate to combine the tables and figures with the manuscript,
+depending on how many tables/figures are included and how long they take to 
+knit.
+
+Go [here](https://rmarkdown.rstudio.com/lesson-1.html) for Rmarkdown 
+instructions.
+
+PDF output from knit Rmarkdown files are not tracked with git in this 
+template. Use the `.gitignore` file if you want to change this.
+
+## References
+
+To save bibliographic information about sources used for your paper, use
+a `.bib` file. This template includes `references.bib`. For information about 
+`.bib` file syntax, go [here](http://web.mit.edu/rsi/www/pdfs/bibtex-format.pdf).
```

Comment:
How have you been outputting .bib files? I've been using Mendeley to manage references and output .bib files

Reply:
I was just writing the .bib file by hand, but I added in a note about Mendeley/Zotero, and I downloaded Mendeley so that I can start using it.

Reply:
Nice, there is a plugin for chrome too that makes it easy to add references from the papers webpage. Also you can create groups to have multiple people add references to a collection.